### DISCUSSION TITLE
'Custom' steps setting: Exclude current date?

### DISCUSSION THREAD

**Discussion by jllyh** [jlai/fitness-dashboard#62] [Ideas]
I used the rolling 28-day average in the former dashboard as a 'steps' target because it was more reachable than a fixed target, and was more likely to encourage gradual but steady improvement.  I'm delighted to see that the 'Custom' setting emulates that to a point; thank you!  One request, though:  Could the default 'Custom' range not include the current date?  When that is included, the average is skewed by the incomplete data for that date. Thanks again!

**Comment by jlai** [comment]
I probably won't change the default custom range, but maybe we could add a dashboard tile that lets you show a stat over a custom number of days
